scripts/build_conflicting_outputs.sh

```diff
@@ -0,0 +1 @@
+flutter pub run build_runner build --delete-conflicting-outputs
```

Comment:
:thinking: Nie wiem czy to aż tak się przyda ale może być

Reply:
Zostawiłbym, bo często zapominam tej komendy, więc sobie tu zerkam